### PR TITLE
[easy-monorepo] Remove EasyStandard and add EasyQuality for quality checks

### PR DESCRIPTION
**Pull request by ERuban**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | yes 
| Deprecations? | no
| Tests pass?   | yes 
| Fixed tickets | -

- Remove EasyStandard
- Remove `quality` dependencies
- Add EasyQuality

We have some `ECS` issues, but I think it will be fixed after updating `ECS` in `easy-quality` package.

### CODE REVIEW

File: packages/EasyApiToken/composer.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
         "auth0/auth0-php": "^5.4",
         "psr/container": "^1.0",
         "laravel/lumen-framework": "^5.5",
-        "symfony/dependency-injection": "5.1.10",
+        "symfony/dependency-injection": "^5.1.10",
```
Comment (roman-eonx) [inline packages/EasyApiToken/composer.json:19]: It was `^4.4 || ^5.1.5` [before the 3.0.5 version](https://github.com/eonx-com/easy-monorepo/blob/3.0.4/packages/EasyApiToken/composer.json). Let's revert it to that restriction back.

File: .github/workflows/php-ci.yml [hunk 2]
```diff
@@ -36,6 +36,8 @@ jobs:
 
             -   uses: eonx-com/actions-composer-install@v1
 
+            -   run: composer install
```
Comment (roman-eonx) [inline .github/workflows/php-ci.yml:39]: If this is needed only for installing EasyQuality, then let's add `--working-dir=.quality`.

File: rector-ci.php [hunk 3]
```diff
@@ -9,7 +9,6 @@
 use Symfony\Component\DependencyInjection\Loader\Configurator\ContainerConfigurator;
 
 return static function (ContainerConfigurator $containerConfigurator): void {
-    $containerConfigurator->import(__DIR__ . '/packages/EasyStandard/config/rector-set.php');
```
Comment (roman-eonx) [inline rector-ci.php:12]: This way we stop using our set in CI. I think we should introduce a new set for this (right in the monorepo) or simply move the contents of that set here.

**Review by roman-eonx — CHANGES_REQUESTED** [review]

File: packages/EasyApiToken/composer.json (hunk 1 above)
Comment (ERuban) [reply packages/EasyApiToken/composer.json]: done

File: .github/workflows/php-ci.yml (hunk 2 above)
Comment (ERuban) [reply .github/workflows/php-ci.yml]: done

File: rector-ci.php (hunk 3 above)
Comment (ERuban) [reply rector-ci.php:12]: added to config, but rector checks are disabled in php-ci

**Review by roman-eonx — APPROVED** [review]

**Review by natepage — APPROVED** [review]
Thank you for that!

**Pull request merged**